lib/utilities/relationship-utils.js

```diff
@@ -69,3 +71,18 @@ function getIncludesArray (query) {
 function shouldIncludeRelationships (method) {
   return method.toLowerCase() === 'get';
 }
+
+function isJSONAPIPagination (req) {
+  return _.isPlainObject(req.query) && req.query.hasOwnProperty('page') &&
+    req.query['page'].hasOwnProperty('size');
+}
+
+/**
+ * Get the pagination data from query parameters
+ */
+function getPagination (query) {
+  return {
+    limit: query.page.size,
+    skip: Math.max(query.page.number - 1, 0) * query.page.size
+  };
```

Comment:
I think perhaps that the best strategy we can apply in regards to pagination syntax is the following:
- use `page[skip]` and `page[limit]` as our query values for pagination since: 
  - we must use the page key according to jsonapi
  - jsonapi is agnostic about what keys we use under page so we can choose whatever page values we like ie. skip and limit
  - loopback uses skip and limit and i believe we should stick with loopbacks conventions wherever its not in conflict with jsonapi
- we should consider supporting the built in loopback pagination syntax simultaneously, ie. `filter[limit]` and `filter[skip]` since:
  - this will be less surprising for anyone already using loopback and adding jsonapi component
  - it shouldn't be any trouble for us to also support this

Thoughts? @taras @maorhayoun